.travis.yml

```diff
@@ -151,12 +151,13 @@ script:
     if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
       do_codecov samples
     fi
+  # Run check-unit{,-coverage} and check-integration.
   - |
     set -ev
     if [ "$TRAVIS_TEST_RESULT" = 0 ]; then
       if [ "$DO_COVERAGE" != "0" ]; then
         travis_fold_start "DO_COVERAGE"
-        (make check-coverage \
+        (make check-unit-coverage \
          && do_codecov unittests \
          && sed -i "1 i\\require('luacov.runner')('"$PWD"/.luacov')" build/awesomerc.lua \
          && BUILD_DIRECTORY="" tests/run.sh \
```

Comment:
This is basically `make check-integration`.  Not sure if we could do `BUILD_DIRECTORY="" make check-integration` here to clarify it?!